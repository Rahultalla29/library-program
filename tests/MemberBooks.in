MEMBER BOOKS
ADD MEMBER Sakura
MEMBER BOOKS 100000
MEMBER BOOKS 100003
ADD COLLECTION sample1.csv
RENT 100000 111111
RENT 100000 111112
MEMBER BOOKS 100000
RELINQUISH 100000 111111
MEMBER BOOKS 100000
RELINQUISH 100000 111112
MEMBER BOOKS 100000
EXIT
