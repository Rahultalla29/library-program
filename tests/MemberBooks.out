user: No members in system.

user: Success.

user: Member not currently renting.

user: No such member in system.

user: 17 books successfully added.

user: Success.

user: Success.

user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)

user: Success.

user: The Hobbit (J.R.R. Tolkien)

user: Success.

user: Member not currently renting.

user: Ending Library process.
